multinode-demo/remote_leader.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+[[ -n $FORCE ]] || exit
+
+mkdir -p ~/.ssh ~/solana ~/.cargo/bin
+sudo apt-get --assume-yes install rsync libssl-dev
+chmod 600 ~/.ssh/authorized_keys ~/.ssh/id_rsa
+
+# Run setup
```

Comment:
stale comment?

Reply:
Actually, useless comment at the wrong spot. Should be few lines down.